Create index.md

minekansu opened the pull request:
## Changes

*Please describe.*

*Add screenshots or screen recordings for visual / UI-focused changes.*

## Checklist

- [ ] Words are spelled using American English
- [ ] Titles are in [sentence case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case)
- [ ] Feature names are in **[sentence case too]([https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case))**
- [ ] Use relative URLs for internal links
- [ ] If I moved a page, I added a redirect in `vercel.json`
- [ ] Remove this template if you're not going to fill it out!

## Article checklist

- [ ] I've added (at least) 3-5 internal links to this new article
- [ ] I've added keywords for this page to the rank tracker in Ahrefs
- [ ] I've checked the preview build of the article
- [ ] The date on the article is today's date
- [ ] I've added this to the relevant "Tutorials and guides" docs page (if applicable)

charlescook-ph reviewed (commented):
Thanks! Some initial thoughts:
- I think you could more explicitly say 'RevOps is the PM for the Sales, Marketing, and Exec teams at PostHog' - we already have a mental model for this with PMs, so I think worth calling out. 
- Small thing - do people call it RevOps? revops? ReVoPs??
- The values are good, but I think each could benefit from a more concrete example? The examples are still generally probably one level of abstraction too high. 
- Under 'vision', this is generally good except the 'brilliant' section which again I think could be one level more specific? One thing we talked about was owning the standardized definitions for things like revenue recognition, revenue retention etc. 
- Probably worth including a 'Responsibilities' type section to clarify what you see revops owning vs. not owning? e.g. the sales ops stuff could be linked to from here

minekansu commented:
> * I think you could more explicitly say 'RevOps is the PM for the Sales, Marketing, and Exec teams at PostHog' - we already have a mental model for this with PMs, so I think worth calling out.

added this at the very beginning

> * Small thing - do people call it RevOps? revops? ReVoPs??

it shall be RevOps ✨ (made it consistent)

> * The values are good, but I think each could benefit from a more concrete example? The examples are still generally probably one level of abstraction too high.

Added some specific examples to each, let me know if this is what you had in mind

> * Under 'vision', this is generally good except the 'brilliant' section which again I think could be one level more specific? One thing we talked about was owning the standardized definitions for things like revenue recognition, revenue retention etc.

made some changes here, I will add more links once I start documenting things like retention defitinitons, forecasting model etc

> * Probably worth including a 'Responsibilities' type section to clarify what you see revops owning vs. not owning? e.g. the sales ops stuff could be linked to from here

added ✅ let me know if i missed anything here!

charlescook-ph commented on contents/handbook/growth/revops/index.md:
```diff
@@ -0,0 +1,119 @@
+## How RevOps works
+RevOps at PostHog is the Product Manager for Sales, Marketing, and Executive teams. Just as PMs help engineering teams build better products by connecting user needs with technical solutions, RevOps helps go to market teams make better decisions by connecting different parts of the business together.
+
+We do this by combining data from marketing, sales, product usage, and customer success to show what's working and what isn't. While individual teams deeply understand their specific areas, we provide insights about how different parts of the business affect each other and help teams see these connections to drive better outcomes.
```
```suggestion
We do this by combining data from marketing, sales, product usage, and customer success to show what's working and what isn't. While individual teams deeply understand their specific areas, we provide insights about how different parts of the business affect each other and help teams see these connections to drive revenue growth for PostHog.
```

charlescook-ph commented on contents/handbook/growth/revops/index.md:
```diff
@@ -0,0 +1,119 @@
+- Finding useful patterns in customer behavior
+- Showing how each team's work affects our customers
+
+Creating views to show total monthly MRR, per product MRR, and per product usage, and filters out the anomalies to make sure make sure analyses are accurate and consistent across teams was one of the early steps we took in this direction. Unifying data from our billing system, salesforce, and vitally to have full context on biggest [gainers](https://us.posthog.com/project/2/insights/ZcynB70W) and ...
```
```suggestion
Creating views to show total monthly MRR, per product MRR, and per product usage, and filters out the anomalies to make sure make sure analyses are accurate and consistent across teams was one of the early steps we took in this direction. Unifying data from our billing system, Salesforce, and Vitally to have full context on biggest [gainers](https://us.posthog.com/project/2/insights/ZcynB70W) and [losers](https://us.posthog.com/project/2/insights/5V4mvAej) queries to show full context on which customers' spend changed the most was another one to simplify access to this info and quickly take action when needed.
```

charlescook-ph commented on contents/handbook/growth/revops/index.md:
```diff
@@ -0,0 +1,119 @@
+
+### What RevOps supports but doesn't own
+
+Revenue reporting and forecasting: RevOps provides recommendations and improvements but does not own implementation and maintenance. This is currently owned by growth team. 
```
```suggestion
Revenue reporting and forecasting: RevOps provides recommendations and improvements but does not own implementation and maintenance. This is currently owned by the Growth team. 
```

charlescook-ph commented on contents/handbook/growth/revops/index.md:
```diff
@@ -0,0 +1,119 @@
+### What RevOps doesn't do
+- Financial accounting (though we work closely with Finance)
+- Individual sales deal management
+- Billing/invoicing platforms, and data infrastructure for revenue reporting owned by growth team
```
```suggestion
- Billing/invoicing platforms, and data infrastructure for revenue reporting owned by the Growth team
```

charlescook-ph commented:
Looks good! Only small nits at this point. Don't forget to add to the Handbook sidebar file as well so it shows up! (I think it's `index.js`)
